client/src/app/users/user-list.component.spec.ts

```diff
@@ -111,10 +129,10 @@ describe('Misbehaving User List', () => {
     // stub UserService for test purposes
     userServiceStub = {
       getUsers: () => new Observable(observer => {
-        observer.error('Error-prone observable');
+        observer.error('getUsers() Observer generates an error');
       }),
       getUsersFiltered: () => new Observable(observer => {
-        observer.error('Error-prone observable');
+        observer.error('getUsersFiltered() Observer generates an error');
       })
     };
```

Comment:
Consider something like:

```ts
userServiceStub = {
  getUsers: () => throwError("Oh no, getUsers() broke!"),
  getUsersFiltered: () => throwError("Oh no, getUsersFiltered() broke!"),
};
```

(You can import the `throwError()` helper function from RxJS 🙂)

Reply:
I changed the message because the observer doesn't generate the error—

the _observable_ (that we're constructing) generates the error, and the _observer_ (that we're talking to) accepts the error.

Reply:
Good point about it not being the `Observer` that broke. Would it be sufficient to say `getUsers()` generated an error?

Reply:
That sounds just fine to me, thanks! ☺️